src/views/HomeView.jsx

```diff
@@ -4,16 +4,17 @@ import { UsersList } from "../state/use-users-list";
 
 const HomeView = ({ uname }) => {
   const { users } = UsersList(uname);
+  var filteredUsers = users.filter(user => user.id !== "undefined");
```

Comment:
don't use `var`, use `const` wherever possible, else, use `let`

Reply:
This is from the master branch, but I will update it.